examples/tests/cancelonsetup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
Also you could probably remove the `skip_outside_setup` as it is a negative test which is not used anywhere...